app/keyConfig-example.js

```diff
@@ -0,0 +1,4 @@
+
+exports.channel : 'your channel here',
+exports.publish_key   : "pub key here",
```

Comment:
@crewapp/devteam We should discuss how we are going to put these keys in the application. It might be best to create a server route to this location